pages/AWorkedExample.html

```diff
@@ -109,7 +109,7 @@ <h4>Second Card</h4>
     the default response like you did for the last card and make sure it
     looks like the following:
   </p>
-  <img src="static/images/tutorial/card2Response1.png" class="tutorial-image">
+  <img src="static/images/tutorial/card2Response1.png" class="tutorial-image" alt="A default response box with the text '[All answers]-> (this will appear when the answer is incorrect)->(try again)' followed by a white text-box to edit and save feedback.">
```

Comment:
I think (not sure) that arrows and square brackets might not be read by screen reader, so I think you can omit these details. At the very least I'm pretty sure that, if the "arrow" is read, it would be read as "hyphen greater-than sign" which is confusing. Try to convey what's needed without too much punctuation.

Reply:
That is an important perspective to have. Will keep that in mind henceforth, thanks!